src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/package-info/test1/package-info.java

```diff
@@ -0,0 +1,5 @@
+/*For test allowing to place annotation before PACKAGE_DEF. Should not cause violation.*/
+@Deprecated
+package com.puppycrawl.tools.checkstyle.checks.whitespace.test1;
+
+import java.lang.System;
```

Comment:
In first test I replaced that import with "import java.lang.Deprecated", but in other cases we don't use annotation, so imports can't be meaningful. When deleting import declaration PACKAGE_DEF shouldn't been checked as last token.